backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Setup.kt

```diff
@@ -1,9 +1,13 @@
 package app.ehrenamtskarte.backend.auth.database
 
+import app.ehrenamtskarte.backend.auth.database.repos.AdministratorsRepository
 import org.jetbrains.exposed.sql.SchemaUtils
 
 fun setupDatabase() {
     SchemaUtils.create(
         Administrators
     )
+    if (AdministratorEntity.count() == 0L) {
+        AdministratorsRepository.insert("admin@example.com", "admin")
```

Comment:
`admin` is the PW here, right? I think we should use bcrypt here to store the passwords.

Reply:
Then we also can set a default password by just using the hashed version of it.

Reply:
`admin` is the pw, yes. `AdminstratorsRepository` does use bcrypt under the hood though...